packages/terra-application-navigation/tests/wdio/application-navigation-spec.js

```diff
@@ -122,6 +122,26 @@ Terra.describeViewports('ApplicationNavigation - Large', ['large'], () => {
 
     Terra.it.validatesElement();
   });
+
+  describe('Should close open popup when custom event is dispatched', () => {
+    before(() => browser.url('/#/raw/tests/terra-application-navigation/application-navigation/application-navigation'));
+
+    it('launch popup', () => {
+      browser.click('[data-application-header-utility="true"]');
+      browser.waitForVisible('[data-terra-popup-content="true"]');
+      browser.pause(50);
```

Comment:
why is the pause needed?

Reply:
I don't believe it is, was leftover when I copied some code from other tests. Fixed here https://github.com/cerner/terra-framework/pull/1255/commits/d1acd6f55250e3e76cca9dae3979ca566a864db9#diff-85a7fe8bb38bbc87cbbac08c28ef3370L132.